packages/calcite-components/conventions/README.md

```diff
@@ -366,6 +370,8 @@ This will create a unique id attribute like `id="calcite-example-51af-0941-54ae-
 
 ## Prerendering and SSR
 
+<!-- TODO: Get suggestions for what this should be replaced with -->
```

Comment:
This section can be removed for now. I will rework it as part of my work for #10237